cmd/workspace/apps/apps.go

```diff
@@ -244,23 +258,25 @@ func newDeploy() *cobra.Command {
 	// TODO: short flags
 	cmd.Flags().Var(&deployJson, "json", `either inline JSON string or @path/to/file.json with request body`)
 
-	cmd.Flags().StringVar(&deployReq.DeploymentId, "deployment-id", deployReq.DeploymentId, `The unique id of the deployment.`)
-	cmd.Flags().Var(&deployReq.Mode, "mode", `The mode of which the deployment will manage the source code. Supported values: [AUTO_SYNC, SNAPSHOT]`)
-	cmd.Flags().StringVar(&deployReq.SourceCodePath, "source-code-path", deployReq.SourceCodePath, `The workspace file system path of the source code used to create the app deployment.`)
+	cmd.Flags().StringVar(&deployReq.AppDeployment.CreateTime, "create-time", deployReq.AppDeployment.CreateTime, `The creation time of the deployment.`)
+	cmd.Flags().StringVar(&deployReq.AppDeployment.Creator, "creator", deployReq.AppDeployment.Creator, `The email of the user creates the deployment.`)
+	// TODO: complex arg: deployment_artifacts
+	cmd.Flags().StringVar(&deployReq.AppDeployment.DeploymentId, "deployment-id", deployReq.AppDeployment.DeploymentId, `The unique id of the deployment.`)
+	cmd.Flags().Var(&deployReq.AppDeployment.Mode, "mode", `The mode of which the deployment will manage the source code. Supported values: [AUTO_SYNC, SNAPSHOT]`)
+	cmd.Flags().StringVar(&deployReq.AppDeployment.SourceCodePath, "source-code-path", deployReq.AppDeployment.SourceCodePath, `The workspace file system path of the source code used to create the app deployment.`)
+	// TODO: complex arg: status
+	cmd.Flags().StringVar(&deployReq.AppDeployment.UpdateTime, "update-time", deployReq.AppDeployment.UpdateTime, `The update time of the deployment.`)
 
-	cmd.Use = "deploy APP_NAME"
+	cmd.Use = "deploy"
```

Comment:
This is a regression. The command is documented like this:
> databricks apps deploy demo --source-code-path /Workspace/Users/...

Reply:
Changed PR to respect path parameters, fixed now